.github/workflows/cross-version-tests.yml

```
@@ -59,7 +59,12 @@ jobs:
           echo "/usr/share/miniconda/bin" >> $GITHUB_PATH
       - name: Update python
         run: |
-          conda install python=3.6
+          if [[ "${{ matrix.package }}" = "scikit-learn" && "${{ matrix.version }}" = "dev" ]]; then
+            python_version=3.7
+          else
+            python_version=3.6
+          fi
+          conda install python=$python_version
```

We could simply use python 3.7 for all tests:

```suggestion
          conda install python=3.7
```

but this might cause other failures so I just use python 3.7 for dev sklearn.